aws/v14.1.1/release.diff

```diff
@@ -0,0 +1,69 @@
+# Generated with:                                                  # Generated with:
+# devctl release create --name 14.1.0 --provider aws --base 14. |  # devctl release create --name 14.1.1 --provider aws --base 14.
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  annotations:                                                       annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp        giantswarm.io/docs: https://docs.giantswarm.io/reference/cp
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v14.1.0                                                 |    name: v14.1.1
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.6.1                                                     version: 1.6.1
+  - componentVersion: 1.1.0                                          - componentVersion: 1.1.0
+    name: cert-manager                                                 name: cert-manager
+    version: 2.4.3                                                     version: 2.4.3
+  - name: chart-operator                                             - name: chart-operator
+    version: 2.13.0                                                    version: 2.13.0
+  - name: cluster-autoscaler                                         - name: cluster-autoscaler
+    version: 1.19.2                                                    version: 1.19.2
+  - componentVersion: 1.6.5                                          - componentVersion: 1.6.5
+    name: coredns                                                      name: coredns
+    version: 1.4.1                                                     version: 1.4.1
+  - componentVersion: 0.7.6                                     |    - name: external-dns
```

Comment:
Component version is missing. It's probably because of devctl discarding it.

Reply:
fixed